src/components/todoInput.js

```diff
@@ -2,36 +2,53 @@ import * as React from 'react';
 import toDoStyles from './todo.css';
 
 export default function (props) {
+  const [text, setText] = React.useState('');
 
-    const [text, setText] = React.useState('');
-
-    const handleKeyDown = (event) => {
-        if (event.key === 'Enter') {
-            onAddClick();
-        }
+  const onAddClick = () => {
+    if (text.length > 0) {
+      props.onAdd(text);
+      setText('');
     }
+  };
 
-    const onChange = (event) =>
-        setText(event.target.value);
-
-
-    const onAddClick = () => {
-        props.onAdd(text);
+  const handleKeyDown = (event) => {
+    // handling 'Enter' keydown events
+    if (event.key === 'Enter') {
+      if (text.length === 0) {
         setText('');
```

Comment:
do we need this?

Reply:
@shubham49  you are right. We don't need this, only one condition would suffice.